spectree/plugins/base.py

```diff
@@ -122,3 +125,65 @@ def get_func_operation_id(self, func: Callable, path: str, method: str):
         if not operation_id:
             operation_id = f"{method.lower()}_{path.replace('/', '_')}"
         return operation_id
+
+
+@dataclass(frozen=True)
+class RawResponsePayload:
+    payload: Union[JsonType, bytes]
+
+
+@dataclass(frozen=True)
+class ResponseValidationResult:
+    validated_response_payload: Any
```

Comment:
maybe `payload` is enough since the outer class name already contains "validation"